java-checks/src/main/java/org/sonar/java/checks/helpers/SyntaxNodePredicates.java

```diff
@@ -47,4 +67,13 @@ public boolean apply(String input) {
     };
   }
 
+  public static Predicate<Type> isType(final String type) {
+    return new Predicate<Type>() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this a named "static" inner class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2694)